index.html

```diff
@@ -166,6 +166,19 @@
 If certain groups are deliberately excluded from using these features, explain
 why.
 
+<aside class="example" title="Excluded Users" id="example-exclusion">
+
+   Example: The [Web GPU specification](https://www.w3.org/TR/webgpu/)
+    provides low-level access to developers, to allow them to render
+    high-performance graphics like advanced 3D-rendering.
+    The GPU for the Web Community Group, have identified that over
+    half a billion users are unable to access WebGPU rendered objects.
```

Comment:
This should ideally say why. I think it's something like

```suggestion
    However, over
    half a billion users [citation needed] don't have the hardware they would need to let websites use the full WebGPU API.
    There is a subset of WebGPU that is compatible with these users' devices, so ...
```

Reply:
The way you've reworded this means the citation required would be the group's source (which I don't have but can have a look), instead of the group. I got the billion figure from the group.